pycqed/measurement/waveform_control/pulsar.py

```diff
@@ -605,7 +624,11 @@ def _hdawg_create_analog_channel_parameters(self, id, name, awg):
         self.add_parameter('{}_internal_modulation'.format(name), 
                            initial_value=False, vals=vals.Bool(),
                            parameter_class=ManualParameter)
-    
+        cmd = self.parameters[
+            f'{awg.name}_awgs_{int((int(id[2:]) - 1) / 2)}_mod_freq']
+        self.add_parameter('{}_mod_freq'.format(name),
```

Comment:
I don't understand the distinction between his parameter and the one added in line 553 but that's probably because of my ignorance of Pulsar and qcodes. But maybe you can add a comment to explain the calculation in 628. Or maybe not if it's again because of said ignorance :p

Reply:
It was an alias name, and I already had a commit to remove the alias again on another branch. I cherry-picked this commit now, so the parameter name based on the awg number (formerly defined in 553) does not exist anymore, and we only have the parameter name based on the channel number now.

Reply:
Yay, my comment wasn't entirely stupid! :D